docs/sidebars.js

```diff
@@ -14,13 +14,19 @@ module.exports = {
       label: "Starter Projects",
       items: [
         'Starter-Projects/starter-projects-basic-prompting',
-        'Starter-Projects/starter-projects-blog-writer',
-        'Starter-Projects/starter-projects-document-qa',
-        'Starter-Projects/starter-projects-memory-chatbot',
-        'Starter-Projects/starter-projects-simple-agent',
         'Starter-Projects/starter-projects-vector-store-rag',
-        'Starter-Projects/starter-projects-sequential-agent',
-        'Starter-Projects/starter-projects-travel-planning-agent',
+        'Starter-Projects/starter-projects-simple-agent',
+      ],
+    },
+    {
+      type: "category",
+      label: "Use Cases",
```

Comment:
I know we discussed adding use-case-based content to the docs. I'm not sure if "Use cases" is the right terminology to use in the docs, though. Maybe try "Tutorials" instead and see how that flows.

Reply:
This PR adds `sidebarCollapsed: true,` to the Docusaurus config.
I think it looks better like this and with the `Tutorials` title.
<img width="347" alt="Screenshot 2024-12-20 at 5 34 15 PM" src="https://github.com/user-attachments/assets/d948d516-5f45-4e11-84ae-991a54e48924" />